app/src/browser/application.ts

```diff
@@ -397,19 +397,19 @@ export default class Application extends EventEmitter {
     });
 
     this.on('application:view-help', () => {
-      const helpUrl = 'http://support.getmailspring.com/hc/en-us';
+      const helpUrl = 'https://community.getmailspring.com/docs';
```

Comment:
I wonder if this should link to https://community.getmailspring.com/categories? It's nice that this docs page has the search bar on it, but it seems to sort the issues beneath the bar by recency which is a bit of a bummer.

Reply:
I still think we should use Docs, as it filters by topics, whereas the main categories does not. Both will sort by recency.